FogCreek/Glitch-Community
# Fifth turret: hookify project editor

## Links
* Remix link: https://fifth-turret.glitch.me

## Changes:
* replace ProjectEditor component with useProjectEditor hook
* clean up project page

## How To Test:
* log into https://fifth-turret.glitch.me
* verify that you can edit a project
